includes/Specials/BrowseData/SemanticResultPrinter.php

```diff
@@ -55,7 +54,7 @@ public function getText( array $displayParameters ) {
 	private static function createGetSmwQueryResult( $res, $num ) {
 		$qr = [];
 		$count = 0;
-		$store = Utils::getSMWStore();
+		$store = smwfGetStore();
```

Comment:
do we need a use `SMW\SQLStore\SQLStore;` here after this change?

Reply:
@gesinn-it-gea Think not, because when the `use SMW\SQLStore\SQLStore;` is included, then the `phpcs` reports the sniff that this is an unused use.